quicly-probes.d

```diff
@@ -20,6 +20,24 @@
  * IN THE SOFTWARE.
  */
 
+/* @appdata
+{
+    "receive": ["bytes"],
+    "packet_received": ["decrypted"],
+    "stream_on_receive": ["src"],
```

Comment:
:+1: These two attributes carry application data in unencrypted form and therefore must be marked as "appdata."